Dockerfile.assisted-installer-controller

```diff
@@ -17,9 +17,8 @@ RUN go mod download
 COPY . .
 RUN TARGETPLATFORM=$TARGETPLATFORM make controller
 
-FROM quay.io/centos/centos:stream8
+FROM quay.io/centos/centos:stream9
 
-RUN yum -y install make gcc unzip wget curl rsync && yum clean all
```

Comment:
no idea why the controller needs all those packages. it doesn't seem like we're calling any of those executables